streams.yml

```diff
@@ -1,3 +1,5 @@
 ---
 rhel8:
-  image: openshift/ose-base:ubi8
+  # the most recent release at present. since we yum update this, maybe it does not need to float.
+  # but it is important that we not build from unreleased builds.
```

Comment:
actually we really don't give a hoot about updating the builder regularly, and we don't release it, so i think just omit this.
```suggestion
```